src/ert/config/gen_data_config.py

```diff
@@ -55,16 +55,24 @@ def from_config_dict(cls, config_dict: ConfigDict) -> Optional[Self]:
             options = option_dict(gen_data, 1)
             name = gen_data[0]
             res_file = options.get("RESULT_FILE")
+            report_steps_value = options.get("REPORT_STEPS", "")
 
             if res_file is None:
                 raise ConfigValidationError.with_context(
                     f"Missing or unsupported RESULT_FILE for GEN_DATA key {name!r}",
                     name,
                 )
+            try:
+                _report_steps: Optional[List[int]] = rangestring_to_list(
+                    report_steps_value
+                )
+            except ValueError as e:
+                raise ConfigValidationError.with_context(
+                    f"The REPORT_STEPS setting: {report_steps_value} is invalid"
```

Comment:
Is it possible to also mention "POLY_RES" in this error message? Can make it easier to navigate to the right spot in the config file if there are many of these.

Reply:
I can add that, but this is how the validation error looks like in the gui, so it seems like the user will get a good pointer to the right spot in the config file already:
![image](https://github.com/user-attachments/assets/f4937351-08b2-471c-8506-d9aa7f729fb3)

Reply:
Here is an alternative where "POLY_RES" is mentioned in the error message: 
![image](https://github.com/user-attachments/assets/fed4ca1d-486b-48a9-9f99-8b93f9980387)

Reply:
Yes, the latter looks very good!